webapp/client/src/components/StepNavButtons.js

```diff
@@ -60,6 +60,44 @@ const Link = styled.a`
   }
 `;
 
+const OutlineLink = styled.a`
+  ${sharedButtonLinkStyle}
+
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+
+  font-weight: var(--font-bold);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--link-color);
+
+  background: white;
+  background-clip: padding-box;
+
+  border: 0;
+  border-radius: 0;
+  border: 1px solid var(--border-color);
```

Comment:
Why we need this double rules here (border)?

Reply:
I added the ${sharedButtonLinkStyle} to the outline link - it should look exactly like the OutlineButton

Reply:
The border information is overwritten for hover, focus etc. That is why I have to set the borders multiple times. Or do you mean something else?

Reply:
Thanks for pointing that out, @punknoir101. I think this is a mistake from the OutlineButton styling and we can delete `border: 0; ` here as well as for the OutlineButton

Reply:
Done!